git_machete/cli.py

```diff
@@ -39,7 +39,7 @@
     ("Build, display and modify the tree of branch dependencies",
      ["add", "anno", "discover", "edit", "status"]),
     ("List, check out and delete branches",
-     ["delete-unmanaged", "go", "is-managed", "list", "show"]),
+     ["clean", "delete-unmanaged", "go", "is-managed", "list", "prune", "show"]),
```

Comment:
`clean` is left out deliberately... it's not yet formally deprecated, but I'll deprecate it soon (and later remove in 3.x->4.0 transition, if it ever happens 😅)